lib/pi_piper/driver.rb

```diff
@@ -0,0 +1,114 @@
+module PiPiper
+  class Driver
+    
+    VERSION = '0.1.0'
+    ALLOW = []
```

Comment:
Yes, you're right, it more an intuition than a need. let's stay pragmatic, I'll remove it.